### ISSUE TITLE
Keyboard navigation unable to reach "do it anyway" checkbox

### ISSUE THREAD

**Issue by Huge** [RabbyHub/Rabby#764]
Tabulator jumps from price settings to cancel and never to the checkbox between them.

**Comment by vvvvvv1vvvvvv** [comment]
Can not understand, can you make any screenshot for it?

**Comment by vvvvvv1vvvvvv** [comment]
Close since no response, let me know for any update and will reopen this issue
